teuthology/task/kube_spray.py

```diff
@@ -0,0 +1,196 @@
+import logging
+import yaml
+
+from . import Task
+from cStringIO import StringIO
```

Comment:
fyi, this only works in Python 2. Is there any plan to support Python 3 in the future?

Reply:
There was one old PR and some work was done to port to python3, not sure how much still applies, but I guess python 3 is must, couple more months before python2 plug is removed :-)

Reply:
There is https://github.com/ceph/teuthology/pull/1362 currently WIP. @kshtsk is working on that and it would be nice if new code is py2 and py3 compatible.